admin/base/urls.py

```diff
@@ -1,5 +1,7 @@
 from django.conf.urls import include, url, patterns
+from django.contrib import admin
 from settings import ADMIN_BASE
+admin.autodiscover()
```

Comment:
What does `autodiscover` help us with? It seems that Django 1.7 and later do this automatically. https://docs.djangoproject.com/en/1.8/ref/contrib/admin/#discovery-of-admin-files